### PR TITLE
[stable-2.8] Don't detect update if vrf not set. (#56235)

### PR DESCRIPTION
**Pull request by Qalthos**
(cherry picked from commit ea41bbc)

Co-authored-by: Nathaniel Case <ncase@redhat.com>

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

eos_eapi

### CODE REVIEW

**Comment by abadger** [comment]
Needs a changelog fragment in order to be merged

File: changelogs/fragments/56235-eapi-vrf.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- eos_eapi: fix idempotency issues when vrf was unspecified.
```
Comment (abadger) [inline changelogs/fragments/56235-eapi-vrf.yaml]: You can quote the line or change the colon into a dash to fix this.

**Comment by pabelanger** [comment]
recheck

**Comment by Qalthos** [comment]
This needs #56134 to get past `% Invalid input (privileged mode required)`

**Comment by Qalthos** [comment]
recheck

**Comment by pabelanger** [comment]
So, for here we could have added
  Depends-on: https://github.com/ansible/ansible/pull/56134

into the commit message, and zuul would have first checked out 56134 then stacked this change on it, resulting in testing before merging og 56134

**Comment by abadger** [comment]
Merged for the 2.8.1 release.

**Pull request merged**